mockgcp/mockcompute/operations.go

```diff
@@ -91,12 +91,12 @@ func (s *computeOperations) startLRO0(ctx context.Context, op *pb.Operation, fqn
 	op.InsertTime = PtrTo(formatTime(now))
 	op.Id = PtrTo(uint64(nanos))
 
-	op.Progress = PtrTo(int32(0))
+	op.Progress = PtrTo(startProgress)
```

Comment:
As we pass in `op`, it might be easier to set it to zero only if Progress is nil.  Avoids the long-list of unnamed args problem!